### ISSUE TITLE
Can translate posts

### ISSUE THREAD

**Issue by danimesq** [mkg20001/ZeroMe#29]
- [ ] Can enable this function
- [ ] Can choose between Google and Bing translator
- [ ] Can click to translate a post
- [ ] Can enable automatic translation

**Comment by mkg20001** [comment]
But Google Translate and Bing are on the clearnet. That needs at least a warning text.

**Comment by danimesq** [comment]
Yes. Because it, also, this function will be disabled by default.

**Comment by mkg20001** [comment]
But we may need an api key. That's were the trouble starts, because if we reach the limit we may need to pay for the key. And the key will be shared in the source code.

**Comment by danimesq** [comment]
Yes. We need a decentralized translator.

**Comment by mkg20001** [comment]
Yes but how do you want to achieve that? Google has thousands of servers and advanced ai technology.
What do we have? A 10mb file limit and tensorflow (python ai software).

**Comment by danimesq** [comment]
If we can form a team, known open-source translator programs can be converted to JS (using emscripten) and then mixed. The merged sites can be the languages. And the translator will communicate with ZeroMe (not all in the same site).
